ibm/ibm-how-to/find-ubuntu-images.rst

```diff
@@ -0,0 +1,32 @@
+Find Ubuntu images on IBM VPC Infrastructure
+============================================
+
+Using Web Console
+-----------------
+
+On your IBM Cloud Console, you can find the latest Ubuntu images by navigating to :guilabel:`VPC Infrastructure` > 
+:guilabel:`Compute`> :guilabel:`Images`> :guilabel:`Stock Images`
+and then searching ``Ubuntu`` in the :guilabel:`Search Images` search bar. 
+
+
+Using IBM Cloud CLI
+------------------- 
+
+For a programmatic way to find the latest Ubuntu images, you can use the IBM Cloud CLI along with grep
+to filter the output.
+
+.. code-block:: bash
+
+    ibmcloud is images --visibility "public" --status available | grep "ubuntu"
+
+
+Find Ubuntu images on IBM Classic Infrastructure
+------------------------------------------------
```

Comment:
Should this `---` be `===` considering the VPC infra one is?

Reply:
good catch!